Weather App/UI/SettingsVC/SettingsViewController.swift

```diff
@@ -8,16 +8,18 @@
 import UIKit
 
 class SettingsViewController: UIViewController {
-    private lazy var settingsView = SettingsView()
+    lazy var settingsView = SettingsView()
```

Comment:
This should be `private`. Check the coordinator comment up.

Reply:
Corrected.